HW34/src/main/java/org/my/homework/ConsoleCartApplication.java

```diff
@@ -0,0 +1,16 @@
+package org.my.homework;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ConsoleCartApplication {
+
+    @Autowired
+    private ConsoleCartService consoleCartService;
+
+    public static void main(String[] args) {
+        SpringApplication.run(ConsoleCartApplication.class, args);
```

Comment:
For this task should not be used Spring boot.

Reply:
Fixed. The starter was implemented by AnnotationConfigApplicationContext